src/lib/snack-bar/simple-snack-bar.scss

```diff
@@ -16,6 +16,14 @@ $mat-snack-bar-button-margin: 48px !default;
   background: none;
   flex-shrink: 0;
   margin-left: $mat-snack-bar-button-margin;
+  line-height: 20px;
```

Comment:
Is this used for centering? Couldn't we use flexbox instead?

Reply:
See other comment